src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -743,7 +743,7 @@ export function onCalendarChange(
           start: new Date(selectedSlot.start),
           end: new Date(selectedSlot.end),
         };
-        const appointmentStart = new Date(appointment.startUtc);
+        const appointmentStart = new Date(appointment.start);
```

Comment:
Need to check but 'appointment.start' should be a Date object (transformed in the transformer).